src/main/java/org/folio/event/handler/HoldingUpdateAsyncRecordHandler.java

```diff
@@ -84,7 +84,10 @@ private Future<List<PoLine>> updatePoLines(InventoryUpdateHolder holder, List<Po
       log.info("updatePoLines:: No POLs were updated for holding, holdingId: {}, POLs retrieved: {}", holder.getHoldingId(), poLines.size());
       return Future.succeededFuture(List.of());
     }
-    return poLinesService.updatePoLines(poLines, conn, holder.getTenantId())
+    // Must pass the correct active tenantId to resolve which schema name is to be used in the poLine update
+    // in a non-ecs environment the active tenantId is the same as the event tenantId
+    // in an ecs environment the active tenantId is the central tenantId
```

Comment:
minor: can be added that only with central ordering feature enabled. Because we can have libraries that use ECS and have orders in each separate tenant